### PR TITLE
Sum of numbers in a string

### PR DESCRIPTION
**Pull request by HariniJeyaraman**
**Fixes issue:**

#4733

**Changes:**

Sum of numbers in a given string in C++

### CODE REVIEW

**Comment by HariniJeyaraman** [comment]
@arnavb Please review the code

File: sumNumbersString.cpp [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// Given a string containing alphanumeric characters, calculate sum of all
+// numbers present in the string.
+#include <cctype>
+#include <cstdlib>
+#include <iostream>
+
+int sum_numbers_string(std::string s) {
```
Comment (arnavb) [inline sumNumbersString.cpp]: Move this file to the correct directory (probably unclassified algorithms). Also change this function name to use lower camel case.

Comment (HariniJeyaraman) [reply sumNumbersString.cpp]: @arnavb done

**Comment by HariniJeyaraman** [comment]
@arnavb Please review the code . Thanks

**Comment by arnavb** [comment]
Last but not least, the folder/filename should be all lowercase with underscore separators.

**Comment by HariniJeyaraman** [comment]
@arnavb done

**Comment by HariniJeyaraman** [comment]
Thanks for all the quick responses !

**Pull request merged**